data-donation-sdk/src/commonMain/kotlin/care/data4life/datadonation/donation/program/model/ProgramModel.kt

```diff
@@ -31,52 +31,60 @@ enum class BlurFunction(val value: String) {
 }
 
 @Serializable(with = BlurFunctionSerializer::class)
-enum class RevocationMode(val value: String) {
+internal enum class RevocationMode(val value: String) {
     DELETE("delete"),
     ANONYMIZE("anonymize")
 }
 
 @Serializable
-internal data class ProgramResourceBlurItem(
+internal data class QuestionnaireResponseItemBlur(
     val linkId: String,
     @SerialName("fn")
     @Contextual
     val function: BlurFunction
 )
 
 @Serializable
-internal data class ProgramResourceBlur(
-    val location: String? = null,
+internal data class ProgramFhirResourceBlur(
```

Comment:
They are not rules...they are DTO's...we make them to Rules (aka Entities)...so I like rather to rename them to avoid confusion.

The 2nd point I do not get...about which domain you are talking here - the program domain or the fhir/anonymization domain?

Reply:
THey are conceptually rules (even if DTOs), they have the same data AND are used all round. The main reason why half of the PR is ununderstandable is this explosion of similar models with inconsistent names, DTOs or not,  so i am asking please make it consistent.
Mostly because if anyone would work with this, they shouldnt need to know the backend models to figure out anything here, and why a lot of models have different naming style and almost the same properties all the time

Reply:
the point here is only one, which is that this by naming "looks like" its a generic rule for fhir resources, but in reality is just for questionnaireResponses, so i was wondering why masking it as a generic one. 
If it is because we need a generic one, we can either make a common interface (even empty) or another sealed class /enum for different types of blurrules